python/cugraph/traversal/bfs.pxd

```diff
@@ -20,11 +20,11 @@ from cugraph.structure.graph_primtypes cimport *
 from libcpp cimport bool
 
 
-cdef extern from "algorithms.hpp" namespace "cugraph":
-
-    cdef void bfs[VT,ET,WT](
+cdef extern from "utilities/cython.hpp" namespace "cugraph::cython":
+    cdef void call_bfs[VT,WT](
```

Comment:
We'd better replace VT, ET, WT with vertex_t, edge_t, weight_t.

Reply:
updated